backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -154,11 +154,17 @@ trait BackendWorkflowInitializationActor extends BackendWorkflowLifecycleActor w
   /**
     * Our predefined sequence to run during preStart
     */
-  final def initSequence(): Future[InitializationSuccess] = for {
-    _ <- validateRuntimeAttributes
-    _ <- validate()
-    data <- beforeAll()
-  } yield InitializationSuccess(data)
+  final def initSequence(): Future[InitializationSuccess] =
+    for {
+      defaultRuntimeAttributes <- coerceDefaultRuntimeAttributes(workflowDescriptor.workflowOptions) |> Future.fromTry _
```

Comment:
`|>` 😍

Reply:
What does this do ? :D

Reply:
It takes the left of `|>` and applies it as an argument to the function to the right.  Eases up on the nesting, which in turn makes things more readable IMO